Views/hello.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html>
+ <head>
+   <meta charset="UTF-8">
+   <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+   <script>
+    $(document).ready(function(){
+        $('#postButton').click(function(){
+            $.post("/hello",
+            {
+              name: "JSONPost"
+            },
+            function(data,status){
+                alert("Data: " + data + "\nStatus: " + status);
+            });
+        });
+    });
+    $(document).ready(function(){
+        $('#putButton').click(function(){
+            $.ajax({
+              url: "/hello",
+              type: 'PUT',
+              data: {
+                name: "JSONPut"
+              },
+              success: function(data,status){
+                  alert("Data: " + data + "\nStatus: " + status);
+              }
+            });
+        });
+    });
+    $(document).ready(function(){
+        $('#deleteButton').click(function(){
+            $.ajax({
+              url: "/hello",
+              type: 'DELETE',
+              success: function(data,status){
+                  alert("Data: " + data + "\nStatus: " + status);
+              }
+            });
+        });
+    });
+</script>
+   <title>Hello: Raw routing</title>
+ </head>
+ <body>
+     <p>What would you like to do?</p>
+     <ul><li>
+         <p>Send an HTTP GET request to &quot;/hello&quot;</p>
```

Comment:
Some of these buttons navigate you to a different page, and some return the response in a pop-up.  Could we be consistent?  The GET and urlencoded POST buttons are the two that direct you to a different page.

It's also not clear what the buttons are doing (manipulating a piece of shared data).  It would be clearer to describe the operations like
> Use HTTP GET on `/hello` to receive a greeting.

> Set the username for the greeting using a JSON-encoded HTTP POST on `/hello`

etc